index.bs

```diff
@@ -1049,6 +1074,203 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
 </details>
 </div>
 
+### {{MLContext/dispatch()}}  ### {#api-mlcontext-dispatch}
+
+Schedules the computational workload of a compiled {{MLGraph}} on the {{MLContext}}'s {{MLContext/[[timeline]]}}.
+
+<div dfn-for="MLContext/dispatch(graph, inputs, outputs)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>graph</dfn>: an {{MLGraph}}. The computational graph to be executed.
+      - <dfn>inputs</dfn>: an {{MLNamedTensors}}. The inputs to the computational graph.
+      - <dfn>outputs</dfn>: an {{MLNamedTensors}}. The outputs to the computational graph.
+
+    **Returns:** {{undefined}}.
+</div>
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>dispatch(|graph|, |inputs|, |outputs|)</dfn> method steps are:
+  </summary>
+    1. Let |allTensors| be a [=/list=] of {{MLTensor}}s consisting of |inputs|'s [=map/values=] [=list/extended=] by |outputs|'s [=map/values=].
+    1. If |allTensors| contains any duplicate [=list/items=], then [=exception/throw=] a {{TypeError}}.
+    1. [=list/For each=] |tensor| of |allTensors|:
+      1. If |tensor|'s {{MLTensor/[[context]]}} is not [=this=], then [=exception/throw=] a {{TypeError}}.
+      1. If |tensor|'s {{MLTensor/[[isDestroyed]]}} is true, then [=exception/throw=] a {{TypeError}}.
+    1. If [=validating tensors with descriptors=] given |inputs| and |graph|'s {{MLGraph/[[inputDescriptors]]}} returns false, then [=exception/throw=] a {{TypeError}}.
+    1. If [=validating tensors with descriptors=] given |outputs| and |graph|'s {{MLGraph/[[outputDescriptors]]}} returns false, then [=exception/throw=] a {{TypeError}}.
+    1. Enqueue the following steps to |graph|'s {{MLGraph/[[context]]}}'s {{MLContext/[[timeline]]}}:
+        1. Issue a compute request to |graph|'s {{MLGraph/[[implementation]]}} given |inputs| and |outputs|.
+
+        Issue(778): Add a mechanism for reporting errors during graph execution.
+
+    1. Return {{undefined}}.
```

Comment:
Nit: per the processing model, this can just be undefined (not the WebIDL type `undefined`). Not a big deal, and I also don't think specs are consistent.